platform/mellanox/non-upstream-patches/README.md

```diff
@@ -0,0 +1,25 @@
+## Mellanox non-upstream linux kernel patches ##
+
+To include non-upstream patches into the sonic-linux image during build time, this folder must contain a patch archive.
+
+### Structure of the patch archive
+
+  1. The tarball should be gzip compressed i.e. of extension .tar.gz
+  2. It should contain a file named series. series should provide an order of which the patches have to be applied
+  3. All the patches should be present in the archive in the same folder where series resides.
+  4. Developers should make sure patches apply cleanly over the existing patches present in the sonic-linux-kernel/ repo.
+
+#### Example
+```
+admin@build-server:/sonic-buildimage/platform/mellanox/non-upstream-patches$ tar -tvf patches.tar.gz

```

Comment:
The verbose output with `-v` is not needed. Or are the permissions and ownership relevant? Then leave it.